src/cudamatrix/cu-kernels-ansi.h

```diff
@@ -30,6 +30,9 @@
 #include "cudamatrix/cu-matrixdim.h"
 
 #if HAVE_CUDA == 1
```

Comment:
OK-- since you'll just be ignoring that handle, remove it from the
interface of the functions.   It's OK if the interface diverges slightly
from the CUBLAS ones, we weren't relying on them being identical.
Dan


On Wed, Apr 13, 2016 at 12:52 PM, Ke Li <notifications@github.com> wrote:

> In src/cudamatrix/cu-kernels-ansi.h
> <https://github.com/kaldi-asr/kaldi/pull/663#discussion_r59613688>:
>
> > @@ -30,6 +30,9 @@
> >  #include "cudamatrix/cu-matrixdim.h"
> >
> >  #if HAVE_CUDA == 1
>
> while if I remove this header , an identifier error occurs:
> ../cudamatrix/cu-kernels-ansi.h(102): error: identifier "cublasHandle_t"
> is undefined
>
> ../cudamatrix/cu-kernels-ansi.h(104): error: identifier "cublasHandle_t"
> is undefined
>
> cu-kernels.cu(746): error: identifier "cublasHandle_t" is undefined
>
> cu-kernels.cu(2360): error: identifier "cublasHandle_t" is undefined
>
> cu-kernels.cu(2364): error: identifier "cublasHandle_t" is undefined
>
> 5 errors detected in the compilation of
> "/tmp/tmpxft_00000df0_00000000-19_cu-kernels.compute_53.cpp1.ii".
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/kaldi-asr/kaldi/pull/663/files/753c6be88875404a0f9a2eb609887dcf67724384..b5be620c91de8ec96b55d993d005e5d307b5d199#r59613688>
>